packages/@react-spectrum/table/src/useTableColumnResize.ts

```diff
@@ -10,13 +10,24 @@
  * governing permissions and limitations under the License.
  */
 
+import {AllHTMLAttributes, RefObject, useRef} from 'react';
+import {ColumnResizeState} from './useTableColumnResizeState';
 import {focusSafely, useFocusable} from '@react-aria/focus';
+import {GridNode} from '@react-types/grid';
 import {mergeProps} from '@react-aria/utils';
 import {useKeyboard, useMove} from '@react-aria/interactions';
 import {useLocale} from '@react-aria/i18n';
-import {useRef} from 'react';
 
-export function useTableColumnResize(state, item, ref): any {
+interface ResizerAria {
+  resizerProps: AllHTMLAttributes<HTMLElement>
```

Comment:
Why is this `AllHTMLAttributes` rather than just `HTMLAttributes`? Should we update other instances to match?
No need to make changes for this PR just wondering.

Reply:
ah, auto complete probably got me, i'll match everything else